Kite SDK/PSPrintSDK/OLKitePrintSDK.m

```diff
@@ -232,10 +232,18 @@ + (NSString *)instagramClientID{
 }
 #endif
 
-+ (void)fetchRemotePlist{
++ (void)fetchRemotePlistsWithCompletionHandler:(void(^)())handler{
+    [OLKitePrintSDK fetchRemotePlistWithURL:@"https://sdk-static.s3.amazonaws.com/kite-ios-remote.plist" completionHandler:^(NSError *error){
+        [OLKitePrintSDK fetchRemotePlistWithURL:[NSString stringWithFormat:@"https://sdk-static.s3.amazonaws.com/kite-ios-remote-%@.plist", [OLKitePrintSDK apiKey]] completionHandler:^(NSError *error2){
```

Comment:
When I was writing it, I didn't think twice about the extra request. But now I remembered that every ms matters to ecommerce. I'll do it that way.